extensions-core/lookups-cached-global/src/main/java/io/druid/server/lookup/namespace/cache/NamespaceExtractionCacheManager.java

```diff
@@ -76,7 +76,7 @@ public NamespaceImplData(
     final Object changeLock = new Object();
     final AtomicBoolean enabled = new AtomicBoolean(false);
     final CountDownLatch firstRun = new CountDownLatch(1);
-    final AtomicReference<String> latestVersion = new AtomicReference<>(null);
+    volatile String latestVersion = null;
```

Comment:
@drcrallen before this variable was CASed, now only written and read. Why it should remain AtomicReference?